### ISSUE TITLE
[iOS, Web] doesn't contain LazyStaggeredGridPositionedItem

### ISSUE THREAD

**Issue by wakaztahir** [JetBrains/compose-multiplatform#2532]
```kotlin
Module "androidx.compose.foundation:foundation" has a reference to symbol [ File '/mnt/agent/work/ba60555d9dde92ae/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ]. Neither the module itself nor its dependencies contain such declaration.

This could happen if the required dependency is missing in the project. Or if there is a dependency of "androidx.compose.foundation:foundation" that has a different version in the project than the version that "androidx.compose.foundation:foundation" was initially compiled with. Please check that the project configuration is correct and has consistent versions of all required dependencies.

The list of "androidx.compose.foundation:foundation" dependencies that may lead to conflicts:
 1. "kotlin" (a library with unknown version)
 2. "androidx.compose.animation:animation" (a library with unknown version)
 3. "androidx.compose.animation:animation-core" (a library with unknown version)
 4. "androidx.compose.foundation:foundation-layout" (a library with unknown version)
 5. "androidx.compose.runtime:runtime" (a library with unknown version)
 6. "androidx.compose.runtime:runtime-saveable" (a library with unknown version)
 7. "androidx.compose.ui:ui" (a library with unknown version)
 8. "androidx.compose.ui:ui-geometry" (a library with unknown version)
 9. "androidx.compose.ui:ui-graphics" (a library with unknown version)
10. "androidx.compose.ui:ui-text" (a library with unknown version)
11. "androidx.compose.ui:ui-unit" (a library with unknown version)
12. "androidx.compose.ui:ui-util" (a library with unknown version)
13. "org.jetbrains.skiko:skiko" (a library with unknown version)
14. "org.jetbrains.kotlin:kotlinx-atomicfu-runtime" (a library with unknown version)
15. "org.jetbrains.kotlinx:atomicfu" (a library with unknown version)
16. "org.jetbrains.kotlinx:kotlinx-coroutines-core" (a library with unknown version)

Project dependencies:
+--- kotlin
+--- Decompose:decompose
|    +--- kotlin
|    +--- Essenty:back-handler
|    |    +--- kotlin
|    |    \--- Essenty:utils-internal
|    |         \--- kotlin
|    +--- Essenty:instance-keeper
|    |    +--- kotlin
|    |    \--- Essenty:utils-internal (*)
|    +--- Essenty:lifecycle
|    |    +--- kotlin
|    |    \--- Essenty:utils-internal (*)
|    +--- Essenty:parcelable
|    |    +--- kotlin
|    |    \--- Essenty:utils-internal (*)
|    +--- Essenty:state-keeper
|    |    +--- kotlin
|    |    +--- Essenty:parcelable (*)
|    |    \--- Essenty:utils-internal (*)
|    \--- Essenty:utils-internal (*)
+--- Essenty:back-handler (*)
+--- Essenty:instance-keeper (*)
+--- Essenty:lifecycle (*)
+--- Essenty:parcelable (*)
+--- Essenty:state-keeper (*)
+--- Essenty:utils-internal (*)
+--- androidx.compose.animation:animation
|    +--- kotlin
|    +--- androidx.compose.animation:animation-core
|    |    +--- kotlin
|    |    +--- androidx.compose.runtime:runtime
|    |    |    +--- kotlin
|    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime
|    |    |    |    \--- kotlin
|    |    |    +--- org.jetbrains.kotlinx:atomicfu
|    |    |    |    \--- kotlin
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core
|    |    |         +--- kotlin
|    |    |         +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |         \--- org.jetbrains.kotlinx:atomicfu (*)
|    |    +--- androidx.compose.runtime:runtime-saveable
|    |    |    +--- kotlin
|    |    |    +--- androidx.compose.runtime:runtime (*)
|    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    +--- androidx.compose.ui:ui
|    |    |    +--- kotlin
|    |    |    +--- androidx.compose.runtime:runtime (*)
|    |    |    +--- androidx.compose.runtime:runtime-saveable (*)
|    |    |    +--- androidx.compose.ui:ui-geometry
|    |    |    |    +--- kotlin
|    |    |    |    +--- androidx.compose.runtime:runtime (*)
|    |    |    |    +--- androidx.compose.ui:ui-util
|    |    |    |    |    \--- kotlin
|    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    +--- androidx.compose.ui:ui-graphics
|    |    |    |    +--- kotlin
|    |    |    |    +--- androidx.compose.runtime:runtime (*)
|    |    |    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    |    |    +--- androidx.compose.ui:ui-unit
|    |    |    |    |    +--- kotlin
|    |    |    |    |    +--- androidx.compose.runtime:runtime (*)
|    |    |    |    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    |    |    |    +--- androidx.compose.ui:ui-util (*)
|    |    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    |    +--- androidx.compose.ui:ui-util (*)
|    |    |    |    +--- org.jetbrains.skiko:skiko
|    |    |    |    |    +--- kotlin
|    |    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    +--- androidx.compose.ui:ui-text
|    |    |    |    +--- kotlin
|    |    |    |    +--- androidx.compose.runtime:runtime (*)
|    |    |    |    +--- androidx.compose.runtime:runtime-saveable (*)
|    |    |    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    |    |    +--- androidx.compose.ui:ui-graphics (*)
|    |    |    |    +--- androidx.compose.ui:ui-unit (*)
|    |    |    |    +--- androidx.compose.ui:ui-util (*)
|    |    |    |    +--- org.jetbrains.skiko:skiko (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    +--- androidx.compose.ui:ui-unit (*)
|    |    |    +--- androidx.compose.ui:ui-util (*)
|    |    |    +--- org.jetbrains.skiko:skiko (*)
|    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    +--- androidx.compose.ui:ui-graphics (*)
|    |    +--- androidx.compose.ui:ui-text (*)
|    |    +--- androidx.compose.ui:ui-unit (*)
|    |    +--- androidx.compose.ui:ui-util (*)
|    |    +--- org.jetbrains.skiko:skiko (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- androidx.compose.foundation:foundation-layout
|    |    +--- kotlin
|    |    +--- androidx.compose.runtime:runtime (*)
|    |    +--- androidx.compose.runtime:runtime-saveable (*)
|    |    +--- androidx.compose.ui:ui (*)
|    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    +--- androidx.compose.ui:ui-graphics (*)
|    |    +--- androidx.compose.ui:ui-text (*)
|    |    +--- androidx.compose.ui:ui-unit (*)
|    |    +--- androidx.compose.ui:ui-util (*)
|    |    +--- org.jetbrains.skiko:skiko (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- androidx.compose.runtime:runtime (*)
|    +--- androidx.compose.runtime:runtime-saveable (*)
|    +--- androidx.compose.ui:ui (*)
|    +--- androidx.compose.ui:ui-geometry (*)
|    +--- androidx.compose.ui:ui-graphics (*)
|    +--- androidx.compose.ui:ui-text (*)
|    +--- androidx.compose.ui:ui-unit (*)
|    +--- androidx.compose.ui:ui-util (*)
|    +--- org.jetbrains.skiko:skiko (*)
|    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    +--- org.jetbrains.kotlinx:atomicfu (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
+--- androidx.compose.animation:animation-core (*)
+--- androidx.compose.foundation:foundation
|    ^^^ This module requires symbol [ File '/mnt/agent/work/ba60555d9dde92ae/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ].
|    +--- kotlin
|    +--- androidx.compose.animation:animation (*)
|    +--- androidx.compose.animation:animation-core (*)
|    +--- androidx.compose.foundation:foundation-layout (*)
|    +--- androidx.compose.runtime:runtime (*)
|    +--- androidx.compose.runtime:runtime-saveable (*)
|    +--- androidx.compose.ui:ui (*)
|    +--- androidx.compose.ui:ui-geometry (*)
|    +--- androidx.compose.ui:ui-graphics (*)
|    +--- androidx.compose.ui:ui-text (*)
|    +--- androidx.compose.ui:ui-unit (*)
|    +--- androidx.compose.ui:ui-util (*)
|    +--- org.jetbrains.skiko:skiko (*)
|    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    +--- org.jetbrains.kotlinx:atomicfu (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
+--- androidx.compose.foundation:foundation-layout (*)
+--- androidx.compose.material3:material3
|    +--- kotlin
|    +--- androidx.compose.animation:animation (*)
|    +--- androidx.compose.animation:animation-core (*)
|    +--- androidx.compose.foundation:foundation (*)
|    |    ^^^ This module requires symbol [ File '/mnt/agent/work/ba60555d9dde92ae/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ].
|    +--- androidx.compose.foundation:foundation-layout (*)
|    +--- androidx.compose.material:material-icons-core
|    |    +--- kotlin
|    |    +--- androidx.compose.runtime:runtime (*)
|    |    +--- androidx.compose.runtime:runtime-saveable (*)
|    |    +--- androidx.compose.ui:ui (*)
|    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    +--- androidx.compose.ui:ui-graphics (*)
|    |    +--- androidx.compose.ui:ui-text (*)
|    |    +--- androidx.compose.ui:ui-unit (*)
|    |    +--- androidx.compose.ui:ui-util (*)
|    |    +--- org.jetbrains.skiko:skiko (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- androidx.compose.material:material-ripple
|    |    +--- kotlin
|    |    +--- androidx.compose.animation:animation (*)
|    |    +--- androidx.compose.animation:animation-core (*)
|    |    +--- androidx.compose.foundation:foundation (*)
|    |    |    ^^^ This module requires symbol [ File '/mnt/agent/work/ba60555d9dde92ae/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ].
|    |    +--- androidx.compose.foundation:foundation-layout (*)
|    |    +--- androidx.compose.runtime:runtime (*)
|    |    +--- androidx.compose.runtime:runtime-saveable (*)
|    |    +--- androidx.compose.ui:ui (*)
|    |    +--- androidx.compose.ui:ui-geometry (*)
|    |    +--- androidx.compose.ui:ui-graphics (*)
|    |    +--- androidx.compose.ui:ui-text (*)
|    |    +--- androidx.compose.ui:ui-unit (*)
|    |    +--- androidx.compose.ui:ui-util (*)
|    |    +--- org.jetbrains.skiko:skiko (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- androidx.compose.runtime:runtime (*)
|    +--- androidx.compose.runtime:runtime-saveable (*)
|    +--- androidx.compose.ui:ui (*)
|    +--- androidx.compose.ui:ui-geometry (*)
|    +--- androidx.compose.ui:ui-graphics (*)
|    +--- androidx.compose.ui:ui-text (*)
|    +--- androidx.compose.ui:ui-unit (*)
|    +--- androidx.compose.ui:ui-util (*)
|    +--- org.jetbrains.skiko:skiko (*)
|    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    +--- org.jetbrains.kotlinx:atomicfu (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
+--- androidx.compose.material:material
|    +--- kotlin
|    +--- androidx.compose.animation:animation (*)
|    +--- androidx.compose.animation:animation-core (*)
|    +--- androidx.compose.foundation:foundation (*)
|    |    ^^^ This module requires symbol [ File '/mnt/agent/work/ba60555d9dde92ae/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ].
|    +--- androidx.compose.foundation:foundation-layout (*)
|    +--- androidx.compose.material:material-icons-core (*)
|    +--- androidx.compose.material:material-ripple (*)
|    +--- androidx.compose.runtime:runtime (*)
|    +--- androidx.compose.runtime:runtime-saveable (*)
|    +--- androidx.compose.ui:ui (*)
|    +--- androidx.compose.ui:ui-geometry (*)
|    +--- androidx.compose.ui:ui-graphics (*)
|    +--- androidx.compose.ui:ui-text (*)
|    +--- androidx.compose.ui:ui-unit (*)
|    +--- androidx.compose.ui:ui-util (*)
|    +--- org.jetbrains.skiko:skiko (*)
|    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    +--- org.jetbrains.kotlinx:atomicfu (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
+--- androidx.compose.material:material-icons-core (*)
+--- androidx.compose.material:material-ripple (*)
+--- androidx.compose.runtime:runtime (*)
+--- androidx.compose.runtime:runtime-saveable (*)
+--- androidx.compose.ui:ui (*)
+--- androidx.compose.ui:ui-geometry (*)
+--- androidx.compose.ui:ui-graphics (*)
+--- androidx.compose.ui:ui-text (*)
+--- androidx.compose.ui:ui-unit (*)
+--- androidx.compose.ui:ui-util (*)
+--- com.juul.indexeddb:core
|    +--- kotlin
|    +--- com.juul.indexeddb:external
|    |    \--- kotlin
|    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    +--- org.jetbrains.kotlinx:atomicfu (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
+--- com.juul.indexeddb:external (*)
+--- com.wakaztahir:timeline-wasm
|    +--- kotlin
|    +--- Decompose:decompose (*)
|    +--- Essenty:back-handler (*)
|    +--- Essenty:instance-keeper (*)
|    +--- Essenty:lifecycle (*)
|    +--- Essenty:parcelable (*)
|    +--- Essenty:state-keeper (*)
|    +--- Essenty:utils-internal (*)
|    +--- androidx.compose.animation:animation (*)
|    +--- androidx.compose.animation:animation-core (*)
|    +--- androidx.compose.foundation:foundation (*)
|    |    ^^^ This module requires symbol [ File '/mnt/agent/work/ba60555d9dde92ae/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ].
|    +--- androidx.compose.foundation:foundation-layout (*)
|    +--- androidx.compose.material3:material3 (*)
|    +--- androidx.compose.material:material (*)
|    +--- androidx.compose.material:material-icons-core (*)
|    +--- androidx.compose.material:material-ripple (*)
|    +--- androidx.compose.runtime:runtime (*)
|    +--- androidx.compose.runtime:runtime-saveable (*)
|    +--- androidx.compose.ui:ui (*)
|    +--- androidx.compose.ui:ui-geometry (*)
|    +--- androidx.compose.ui:ui-graphics (*)
|    +--- androidx.compose.ui:ui-text (*)
|    +--- androidx.compose.ui:ui-unit (*)
|    +--- androidx.compose.ui:ui-util (*)
|    +--- com.juul.indexeddb:core (*)
|    +--- com.juul.indexeddb:external (*)
|    +--- io.ktor:ktor-client-auth
|    |    +--- kotlin
|    |    +--- io.ktor:ktor-client-core
|    |    |    +--- kotlin
|    |    |    +--- io.ktor:ktor-events
|    |    |    |    +--- kotlin
|    |    |    |    +--- io.ktor:ktor-http
|    |    |    |    |    +--- kotlin
|    |    |    |    |    +--- io.ktor:ktor-io
|    |    |    |    |    |    +--- kotlin
|    |    |    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    |    |    +--- io.ktor:ktor-utils
|    |    |    |    |    |    +--- kotlin
|    |    |    |    |    |    +--- io.ktor:ktor-io (*)
|    |    |    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    |    +--- io.ktor:ktor-io (*)
|    |    |    |    +--- io.ktor:ktor-utils (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    +--- io.ktor:ktor-http (*)
|    |    |    +--- io.ktor:ktor-io (*)
|    |    |    +--- io.ktor:ktor-serialization
|    |    |    |    +--- kotlin
|    |    |    |    +--- io.ktor:ktor-http (*)
|    |    |    |    +--- io.ktor:ktor-io (*)
|    |    |    |    +--- io.ktor:ktor-utils (*)
|    |    |    |    +--- io.ktor:ktor-websockets
|    |    |    |    |    +--- kotlin
|    |    |    |    |    +--- io.ktor:ktor-http (*)
|    |    |    |    |    +--- io.ktor:ktor-io (*)
|    |    |    |    |    +--- io.ktor:ktor-utils (*)
|    |    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    +--- io.ktor:ktor-utils (*)
|    |    |    +--- io.ktor:ktor-websocket-serialization
|    |    |    |    +--- kotlin
|    |    |    |    +--- io.ktor:ktor-http (*)
|    |    |    |    +--- io.ktor:ktor-io (*)
|    |    |    |    +--- io.ktor:ktor-serialization (*)
|    |    |    |    +--- io.ktor:ktor-utils (*)
|    |    |    |    +--- io.ktor:ktor-websockets (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    |    +--- io.ktor:ktor-websockets (*)
|    |    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    +--- io.ktor:ktor-events (*)
|    |    +--- io.ktor:ktor-http (*)
|    |    +--- io.ktor:ktor-io (*)
|    |    +--- io.ktor:ktor-serialization (*)
|    |    +--- io.ktor:ktor-utils (*)
|    |    +--- io.ktor:ktor-websocket-serialization (*)
|    |    +--- io.ktor:ktor-websockets (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- io.ktor:ktor-client-content-negotiation
|    |    +--- kotlin
|    |    +--- io.ktor:ktor-client-core (*)
|    |    +--- io.ktor:ktor-events (*)
|    |    +--- io.ktor:ktor-http (*)
|    |    +--- io.ktor:ktor-io (*)
|    |    +--- io.ktor:ktor-serialization (*)
|    |    +--- io.ktor:ktor-utils (*)
|    |    +--- io.ktor:ktor-websocket-serialization (*)
|    |    +--- io.ktor:ktor-websockets (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- io.ktor:ktor-client-core (*)
|    +--- io.ktor:ktor-client-js
|    |    +--- kotlin
|    |    +--- io.ktor:ktor-client-core (*)
|    |    +--- io.ktor:ktor-events (*)
|    |    +--- io.ktor:ktor-http (*)
|    |    +--- io.ktor:ktor-io (*)
|    |    +--- io.ktor:ktor-serialization (*)
|    |    +--- io.ktor:ktor-utils (*)
|    |    +--- io.ktor:ktor-websocket-serialization (*)
|    |    +--- io.ktor:ktor-websockets (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- io.ktor:ktor-events (*)
|    +--- io.ktor:ktor-http (*)
|    +--- io.ktor:ktor-io (*)
|    +--- io.ktor:ktor-serialization (*)
|    +--- io.ktor:ktor-serialization-kotlinx
|    |    +--- kotlin
|    |    +--- io.ktor:ktor-http (*)
|    |    +--- io.ktor:ktor-io (*)
|    |    +--- io.ktor:ktor-serialization (*)
|    |    +--- io.ktor:ktor-utils (*)
|    |    +--- io.ktor:ktor-websockets (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-serialization-core
|    |         \--- kotlin
|    +--- io.ktor:ktor-serialization-kotlinx-json
|    |    +--- kotlin
|    |    +--- io.ktor:ktor-http (*)
|    |    +--- io.ktor:ktor-io (*)
|    |    +--- io.ktor:ktor-serialization (*)
|    |    +--- io.ktor:ktor-serialization-kotlinx (*)
|    |    +--- io.ktor:ktor-utils (*)
|    |    +--- io.ktor:ktor-websockets (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    |    +--- org.jetbrains.kotlinx:kotlinx-serialization-core (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-serialization-json
|    |         +--- kotlin
|    |         \--- org.jetbrains.kotlinx:kotlinx-serialization-core (*)
|    +--- io.ktor:ktor-utils (*)
|    +--- io.ktor:ktor-websocket-serialization (*)
|    +--- io.ktor:ktor-websockets (*)
|    +--- org.jetbrains.compose.web:internal-web-core-runtime
|    |    +--- kotlin
|    |    +--- androidx.compose.runtime:runtime (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- org.jetbrains.compose.web:web-core
|    |    +--- kotlin
|    |    +--- androidx.compose.runtime:runtime (*)
|    |    +--- org.jetbrains.compose.web:internal-web-core-runtime (*)
|    |    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    |    +--- org.jetbrains.kotlinx:atomicfu (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- org.jetbrains.skiko:skiko (*)
|    +--- org.jetbrains:markdown
|    |    \--- kotlin
|    +--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
|    +--- org.jetbrains.kotlinx:atomicfu (*)
|    +--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
|    +--- org.jetbrains.kotlinx:kotlinx-datetime
|    |    +--- kotlin
|    |    \--- org.jetbrains.kotlinx:kotlinx-serialization-core (*)
|    +--- org.jetbrains.kotlinx:kotlinx-serialization-core (*)
|    \--- org.jetbrains.kotlinx:kotlinx-serialization-json (*)
+--- io.ktor:ktor-client-auth (*)
+--- io.ktor:ktor-client-content-negotiation (*)
+--- io.ktor:ktor-client-core (*)
+--- io.ktor:ktor-client-js (*)
+--- io.ktor:ktor-events (*)
+--- io.ktor:ktor-http (*)
+--- io.ktor:ktor-io (*)
+--- io.ktor:ktor-serialization (*)
+--- io.ktor:ktor-serialization-kotlinx (*)
+--- io.ktor:ktor-serialization-kotlinx-json (*)
+--- io.ktor:ktor-utils (*)
+--- io.ktor:ktor-websocket-serialization (*)
+--- io.ktor:ktor-websockets (*)
+--- org.jetbrains.compose.web:internal-web-core-runtime (*)
+--- org.jetbrains.compose.web:web-core (*)
+--- org.jetbrains.skiko:skiko (*)
+--- org.jetbrains:markdown (*)
+--- org.jetbrains.kotlin:kotlinx-atomicfu-runtime (*)
+--- org.jetbrains.kotlinx:atomicfu (*)
+--- org.jetbrains.kotlinx:kotlinx-coroutines-core (*)
+--- org.jetbrains.kotlinx:kotlinx-datetime (*)
+--- org.jetbrains.kotlinx:kotlinx-serialization-core (*)
\--- org.jetbrains.kotlinx:kotlinx-serialization-json (*)

(*) - dependencies omitted (listed previously)
```

**Comment by wakaztahir** [comment]
```kotlin
implementation(compose.web.core)
implementation(compose.runtime)
implementation(compose.material)
implementation(compose.material3)
```

**Comment by wakaztahir** [comment]
Removing the dependency on material doesn't fix the problem

**Comment by wakaztahir** [comment]
I went back to 1.2.1 but now I'm getting onWasmReady isn't defined in the console
https://github.com/JetBrains/compose-jb/issues/2273
It says it fails the first time but on my computer I ran multiple times but its still undefined !

**Comment by wakaztahir** [comment]
Ok I had to had <scrip src="skiko.js"></script> but it still doesn't work yet :)

**Comment by wakaztahir** [comment]
Ok now it works , forgot adding , #ComposeTarget canvas in index.html

**Comment by wakaztahir** [comment]
lovely , I can't believe it , material3 in the browser 🥇

**Comment by wakaztahir** [comment]
some things though 
1 - painterResource is missing
2 - canvas width and height changing doesn't update the width and height of the app , need api to change it while its running

me using this code to make it like full screen

```javascript
<script>
    let targetCanvas = document.getElementById("ComposeTarget")
    targetCanvas.width = document.documentElement.clientWidth
    targetCanvas.height = document.documentElement.clientHeight
</script>
```

3 - isSystemInDarkTheme doesn't work

**Comment by wakaztahir** [comment]
I also had to hard code my strings , like since coming from Android , just imported the strings.xml into Kotlin file and made it an object , I think sharing Resources should be solved first , Thanks for this incredible library though !

**Comment by eygraber** [comment]
Check out https://github.com/mipastgt/JavaForumStuttgartTalk2022/blob/5193c1a2bd996b6f175243c143aa5140fb3e8b82/PolySpiralMpp/src/jsMain/kotlin/BrowserViewportWindow.kt to get a resizable window

**Comment by mbakgun** [comment]
compose for iOS(Alpha) currently doesn't have this class.

`error: Module "org.jetbrains.compose.foundation:foundation (org.jetbrains.compose.foundation:foundation-uikitsimarm64)" has a reference to symbol [ File '/opt/buildAgent/work/172e546b22c60d2b/compose/frameworks/support/compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt' <- androidx.compose.foundation.lazy.staggeredgrid/LazyStaggeredGridPositionedItem|null[0] ]. Neither the module itself nor its dependencies contain such declaration. `

**Comment by zacharee** [comment]
A workaround(?) for iOS at least seems to be to use Gradle 8.0, Kotlin 1.8.10, and Compose 1.5.0-dev1049.

Kotlin 1.8.20 in this configuration causes the following compilation error:

```
error: java.lang.IllegalStateException: Suspend functions should be lowered out at this point, but FUN LOCAL_FUNCTION_FOR_LAMBDA name:<anonymous> visibility:local modality:FINAL <> ($receiver:androidx.compose.foundation.gestures.PressGestureScope, it:androidx.compose.ui.geometry.Offset) returnType:kotlin.Unit [suspend] is still here
```

which is supposedly fixed in Kotlin 1.9.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2532">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
